### PR TITLE
turn on objcopy by default for all targets

### PR DESCRIPTION
**Pull request by nickdesaulniers**

### CODE REVIEW

File: driver.sh [hunk 1]
```diff
@@ -184,6 +183,14 @@ check_dependencies() {
   fi
   check_ar_version
   ${AR} --version
+
+  if [[ -z "${OBJCOPY:-}" ]]; then
+    for OBJCOPY in llvm-objcopy-9 llvm-objcopy-8 llvm-objcopy-7 llvm-objcopy "${CROSS_COMPILE:-}"objcopy; do
+      command -v ${OBJCOPY} 2>/dev/null && break
+    done
+  fi
+  check_ar_version
```
Comment (nathanchance) [inline driver.sh]: Should be `check_objcopy_version` :)

**Comment by nickdesaulniers** [comment]
rerunning, as I think https://github.com/ClangBuiltLinux/linux/issues/435 is now resolved.

**Comment by nathanchance** [comment]
I don't think we don't have that patch in our clang build yet (version string says r360126, the patch is r360233). It should be there on the next refresh.

**Comment by nickdesaulniers** [comment]
@nathanchance I think we can merge this. Pre-existing failures are related to lld.

**Comment by nathanchance** [comment]
Hmmm interesting, how did this pass? Clang wasn't refreshed last night according to apt.llvm.org so nothing should have changed...

**Comment by nickdesaulniers** [comment]
oh, we probably fail the `-n` lld check for arm64 before we hit the objcopy issue.

**Pull request merged**